Corona-Warn-App/src/main/java/de/rki/coronawarnapp/risk/DefaultRiskLevels.kt

```diff
@@ -254,7 +255,7 @@ class DefaultRiskLevels @Inject constructor() : RiskLevels {
 
         // 2. Determine `Normalized Time per Date`
         val normalizedTime = exposureWindowsAndResultForDate.values
-            .sumOf { it.normalizedTime }
+            .sumOf { it.normalizedTime.roundToDecimal(decimalPlacesNumber = 1u) }
```

Comment:
I think it might also be more appropriate to only round the sum and not individual operands.